tajo-plan/src/main/java/org/apache/tajo/plan/util/PartitionFilterAlgebraVisitor.java

```diff
@@ -150,8 +151,7 @@ public Expr visitTimestampLiteral(Object ctx, Stack<Expr> stack, TimestampLitera
       DateTimeUtil.toUTCTimezone(tm, tz);
 
       sb.append("?").append(" )");
-      Timestamp timestamp = new Timestamp(DateTimeUtil.julianTimeToJavaTime(DateTimeUtil.toJulianTimestamp(tm)));
-      parameters.add(new Pair(Type.TIMESTAMP, timestamp));
+      parameters.add(new Pair(Type.TIMESTAMP, new TimestampDatum(DateTimeUtil.toJulianTimestamp(tm))));
```

Comment:
This code seem to change the timestamp to a julian time. Could you check if it is valid? We need to make the partitioned table compatible with Apache Hive. Hive does not appear to use a julian time.